Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scalyr/scalyr-agent
+FROM scalyr/scalyr-docker-agent
```

Comment:
it's just an extension for scalyr-agent?

Reply:
yes, the problem here is that scalyr-agent hasn't been updated for about 2 years, scalyr-docker-agent is an updated version of that image